PleaseStandUp.js

```diff
@@ -21,4 +34,19 @@ class PleaseStandUp {
         return -1;
     }
 
-}
\ No newline at end of file
+}
+
+timeConvert(numOfMins)
+// take the num value and then divide it by 60 min to give the hours in the num value
+
+// if there is a remainder from the hour value, then the remainder of num will populate the minute value
+
+// if there is a remainder from the minute value, then remainder will populate the seconds value
```

Comment:
Get decimal of input and convert to seconds (ie. 63.5 minutes => .5 minutes => 30 sec)